### DISCUSSION TITLE
Meta needs property for open graph support

### DISCUSSION THREAD

**Discussion by ShayBox** [DioxusLabs/dioxus#1859] [Q&A] [answered]
https://github.com/DioxusLabs/dioxus/blob/master/packages/html/src/elements.rs#L440-L445  
https://ogp.me/
```html
<meta property="og:title" content="The Rock" />
<meta property="og:type" content="video.movie" />
<meta property="og:url" content="https://www.imdb.com/title/tt0117500/" />
<meta property="og:image" content="https://ia.media-imdb.com/images/rock.jpg" />
```
```rust
meta { property: "og:type", content: "website" }
```

**Comment by ealmloff** [comment, marked as answer]
It doesn't look like the `property` attribute is [officially in the spec](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-meta-element.html), but you can always add [custom attributes](https://dioxuslabs.com/learn/0.4/reference/rsx#custom-attributes):
```rust
meta { "property": "og:type", content: "website" }
```

**Reply by ShayBox** [reply]
Thanks 👍 I didn't know that was possible
